Task2/getMiddle.js

```diff
@@ -0,0 +1,12 @@
+
+/*You are going to be given a word. Your job is to return the middle character of the word. If the word's length is odd, return the middle character. 
+If the word's length is even, return the middle 2 characters.
+*/
+
+function getMiddle(s)
+{
+  if(s.length%2===0)
+    return s.substring(s.length/2-1,s.length/2+1);
```

Comment:
Please use a variable to decide on where to start the substring, and write a single `s.substring` statement for better readability.